nodejs/week1/src/backend/index.js

```diff
@@ -0,0 +1,85 @@
+const express = require("express");
+
+const app = express();
+const port = 3000;
+
+///meals///
+/*app.get("/meals", (request, response) => {
+  let mealsData = getDataFromJson(meals);
+  let reviewsData = getDataFromJson(reviews);
+  let mealsWithReviews = mealsData.map((meal) => {
+    meal.review = reviewsData.filter((review) => review.id === meal.id);
+    return meal;
+  });
+  response.send(mealsWithReviews);
+});*/
```

Comment:
Remove dead code. Commented out code is like weed in a garden. Remove it. If you want to look at some older code you can always use source control for that.

Reply:
the reason why I have left it was a task: Setting up the routes. Inside the index.js file create the following routes using express. Then we used it directly in the meals.js file. But, of course, I will remove it from index.js